hare3/hare_test.go

```diff
@@ -699,6 +699,61 @@ func TestHandler(t *testing.T) {
 	})
 }
 
+func TestHandlerOwnEquivocation(t *testing.T) {
```

Comment:
sorry. i couldn't find a way to add it in TestHandler. 
i didn't know how to advance to a round that the node is eligible again

Reply:
i really prefer not to add such tests, i can't work with them

Reply:
why not to extend TestHandler? this is almost the same as existing test, but with exception of global state

Reply:
i think the way to go to introduce such test is to make initialization reusable, instead of copying it